src/Domain.LinnApps/Parts/Part.cs

```diff
@@ -0,0 +1,165 @@
+
+        public decimal? CurrencyUnitPrice { get; set; }
+
+        public decimal? BaseUnitPrice { get; set; }
+
+        public decimal? MaterialPrice { get; set; }
+
+        public decimal? LabourPrice { get; set; }
+
+        public decimal? CostingPrice { get; set; }
+
+        public string OrderHold { get; set; }
+
+        public decimal? NonForecastRequirement { get; set; }
+
+        public decimal? OneOffRequirement { get; set; }
+
+        public decimal? SparesRequirement { get; set; }
+
+        public string IgnoreWorkstationStock { get; set; }
+
+        public int? ImdsIdNumber { get; set; }
+
+        public decimal? ImdsWeight { get; set; }
+
+        public string QcOnReceipt { get; set; }
+
+        public string QcInformation { get; set; }
+
+        public string RawOrFinished { get; set; }
+
+        public int? OurInspectionWeeks { get; set; }
+
+        public int? SafetyWeeks { get; set; }
+
+        public string RailMethod { get; set; }
+
+        public decimal? MinStockRail { get; set; }
+
+        public decimal? MaxStockRail { get; set; }
+
+        public string SecondStageBoard { get; set; }
+
+        public string SecondStageDescription { get; set; }
+
+        public string TqmsCategoryOverride { get; set; }
+
+        public string StockNotes { get; set; }
+
+        public DateTime? DateCreated { get; set; }
+
+        public DateTime? DateLive { get; set; }
+        
+        public DateTime? DatePhasedOut { get; set; }
+        
+        public string ReasonPhasedOut { get; set; }
+
+        public string ScrapOrConvert { get; set; }
+
+        public string PurchasingPhaseOutType { get; set; }
+
+        public string PlannerStory { get; set; }
+
+        public DateTime? DateDesignObsolete { get; set; }
+        
+        public string LibraryName { get; set; }
+
+        public string LibraryRef { get; set; }
+
+        public string FootprintRef1 { get; set; }
+
+        public string FootprintRef2 { get; set; }
+
+        public string FootprintRef3 { get; set; }
+
+        public string AltiumType { get; set; }
+
+        public string ManufacturersPartNumber { get; set; }
+
+        public string DataSheetPdfPath { get; set; }
+
+        public int? TemperatureCoefficient { get; set; }
+
+        public string Device { get; set; }
+
+        public string Construction { get; set; }
+
+        public string Dielectric { get; set; }
+
+        public int? CapNegativeTolerance { get; set; }
+
+        public int? CapPositiveTolerance { get; set; }
+
+        public decimal? CapVoltageRating { get; set; }
+
+        public string Frequency { get; set; }
+
+        public string FrequencyLabel { get; set; }
+
+        public string SimKind { get; set; } // todo - delete if this never ends up being used
+
+        public string SimSubKind { get; set; } // todo - delete if this never ends up being used
+
+        public string SimModelName { get; set; } // todo - delete if this never ends up being used
```

Comment:
yeah these weird fields were added for ECIT and possibly shouldn't _really_ be on the part, alhtough I couldnt think of a better way. Might as well leave them in here for now though since I do intend to move that code from stores over to stores2 at some point